src/components/NavBar/NavBar.css

```diff
@@ -0,0 +1,49 @@
+:root {
+    --primary-color: #2563EB;
+    --secondary-color: #eff6ff;
+    --gold: rgb(253, 196, 52);
+    --white: #ffff;
+    --black: rgba(0, 0, 0, 0.8);
+}
+
```

Comment:
Since this is repeating, you can create a common css file for this and import that directly here.